### PR TITLE
[Doc] 3.2.4 release notes

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
## What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Review by amber-create — DISMISSED** [review]

File: docs/zh/release_notes/release-3.2.md [hunk 1]
```diff
@@ -4,6 +4,51 @@ displayed_sidebar: "Chinese"
...
 # StarRocks version 3.2
 
+## 3.2.4
+
+发布日期：2024 年 3 月 8 日
+
+### 新增特性
+
+- 存算分离集群中的云原生主键表支持 Size-tiered 模式 Compaction，以减轻导入较多小文件时 Compaction 的写放大问题。[#41034](https://github.com/StarRocks/starrocks/pull/41034)
+- Storage Volume 支持 HDFS 的参数化配置，包括支持 Simple 认证方式支持配置 username，Kerberos 认证，NameNode HA，以及 ViewFS。
```
Comment (evelynzhaojie) [inline docs/zh/release_notes/release-3.2.md:14]: 支持 Simple 认证方式支持配置 --- 第一个支持是否多余

File: docs/zh/release_notes/release-3.2.md [hunk 2]
```diff
@@ -4,6 +4,51 @@ displayed_sidebar: "Chinese"
...
 
+## 3.2.4
+
+发布日期：2024 年 3 月 8 日
+
+### 新增特性
+
+- 存算分离集群中的云原生主键表支持 Size-tiered 模式 Compaction，以减轻导入较多小文件时 Compaction 的写放大问题。[#41034](https://github.com/StarRocks/starrocks/pull/41034)
+- Storage Volume 支持 HDFS 的参数化配置，包括支持 Simple 认证方式支持配置 username，Kerberos 认证，NameNode HA，以及 ViewFS。 
+- 新增日期函数 `milliseconds_dff`。[#38171](https://github.com/StarRocks/starrocks/pull/38171)
```
Comment (evelynzhaojie) [inline docs/zh/release_notes/release-3.2.md:15]: milliseconds_diff

File: docs/zh/release_notes/release-3.2.md [hunk 3]
```diff
@@ -4,6 +4,51 @@ displayed_sidebar: "Chinese"
...
+### 行为变更
+
+- `cbo_decimal_cast_string_strict` 用于优化器控制 DECIMAL 类型转为 STRING 类型的行为。默认值是 `true`，即执行严格转换（按 Scale 截断补 `0`）。在历史版本中没有严格按照 DECIMAL 类型进行补齐，从而在 DECIMAL 与 STRING 类型进行比等时会产生不同效果。[#40619](https://github.com/StarRocks/starrocks/pull/40619)
+- Iceberg Catalog 的参数 `enable_iceberg_metadata_cache` 默认值改为 `false`。在 v3.2.1 到 v3.2.3 版本中，此参数统一默认为 `true`，即默认缓存 Iceberg 的元数据信息。在 v3.2.4 及以上版本中，如果 Iceberg Catalog 元数据服务为 HMS，此参数默认值变更为 `false`，如果元数据服务为 Glue，此参数默认值仍为 `true`。[#41826](https://github.com/StarRocks/starrocks/pull/41826)
+- 修改发起物化视图刷新任务的用户，从原本的 `root` 用户变成创建物化视图的用户，已有的物化视图不受影响。[#40670](https://github.com/StarRocks/starrocks/pull/40670)  
+- 常量和字符串类型的列进行比较时，默认按字符串进行比较，用户可以通过设置变量 `cbo_eq_base_type` 来调整默认行为。将 `cbo_eq_base_type` 设置为 `decimal` 可以改为按数值进行比较。[#40619](https://github.com/StarRocks/starrocks/pull/40619)
+
+### 功能优化
+
+- 存算分离架构中，支持将数据分区存储于 S3 存储桶中的不同路径中，分区路径使用统一前缀。此举可以提升 StarRocks 对 S3 的文件的读写访问效率。[#41627](https://github.com/StarRocks/starrocks/pull/41627)
```
Comment (evelynzhaojie) [inline docs/zh/release_notes/release-3.2.md:31]: StarRocks 对 S3 的文件  --- 删除 的

File: docs/en/release_notes/release-3.2.md [hunk 4]
```diff
@@ -4,6 +4,50 @@ displayed_sidebar: "English"
...
 # StarRocks version 3.2
 
+## 3.2.4
+
+Release date: March 8, 2024
+
+### New Features
+
+- Cloud-native Primary Key tables in shared-data clusters support Size-tiered Compaction to reduce the write I/O amplification. [#41034](https://github.com/StarRocks/starrocks/pull/41034)
+- Added the date function `milliseconds_dff`. [#38171](https://github.com/StarRocks/starrocks/pull/38171)
```
Comment (evelynzhaojie) [inline docs/en/release_notes/release-3.2.md:14]: diff

File: docs/en/release_notes/release-3.2.md [hunk 5]
```diff
@@ -4,6 +4,50 @@ displayed_sidebar: "English"
...
+
+### Improvements
+
+- Shared-data StarRocks clusters with an AWS S3 Storage Volume support partitioning data in uniformly prefixed sub-paths of the bucket, improving the read and write efficiency on data files in S3. [#41627](https://github.com/StarRocks/starrocks/pull/41627)
+- StarRocks supports using the parameter `s3_compatible_fs_list` to specify which S3-compatible object storage can be accessed via AWS SDK, and supports using the parameter `fallback_to_hadoop_fs_list` to specify non-S3-compatible object storages that require access via HDFS Schema (this method necessitates the use of vendor-provided JAR packages). [#41123](https://github.com/StarRocks/starrocks/pull/41123)
+- The compatibility with Trino's SQL statement syntax is optimized to support the syntax conversion from the following functions of Trino: `current_catalog`, `current_schema`, `to_char`, `from_hex`, `to_date`, `to_timestamp`, and `index`. [#41217](https://github.com/StarRocks/starrocks/pull/41217) [#41319](https://github.com/StarRocks/starrocks/pull/41319)[ ](https://github.com/StarRocks/starrocks/pull/40837)[#40803](https://github.com/StarRocks/starrocks/pull/40803)
```
Comment (evelynzhaojie) [inline docs/en/release_notes/release-3.2.md:32]: [ ](https://github.com/StarRocks/starrocks/pull/40837) --- need the PR ID in []

Comment (evelynzhaojie) [reply docs/en/release_notes/release-3.2.md:32]: - necessitates --> requires
- Optimized compatibility with Trino. Supports syntax conversion from the following Trino functions: `current_catalog`, `current_schema`, `to_char`, `from_hex`, `to_date`, `to_timestamp`, and `index`.

File: docs/en/release_notes/release-3.2.md [hunk 6]
```diff
@@ -4,6 +4,50 @@ displayed_sidebar: "English"
...
+- The user who can refresh materialized views is changed from the `root` user to the user who creates the materialized views. This change does not affect existing materialized views. [#40670](https://github.com/StarRocks/starrocks/pull/40670)  
+- By default, when comparing columns of constant and string types, StarRocks compares them as strings. Users can use the session variable `cbo_eq_base_type` to adjust the rule used for the comparison. For example, users can set `cbo_eq_base_type` to `decimal`, and StarRocks then compares the columns as numeric values. [#40619](https://github.com/StarRocks/starrocks/pull/40619)
+
+### Improvements
+
+- Shared-data StarRocks clusters with an AWS S3 Storage Volume support partitioning data in uniformly prefixed sub-paths of the bucket, improving the read and write efficiency on data files in S3. [#41627](https://github.com/StarRocks/starrocks/pull/41627)
```
Comment (evelynzhaojie) [inline docs/en/release_notes/release-3.2.md:30]: 断一下句，更好理解。Shared-data StarRocks clusters with an AWS S3 Storage Volume support partitioning data to different paths of an S3 bucket. These paths use a uniform prefix. This improves the read and write efficiency on data files in S3.

File: docs/zh/release_notes/release-3.2.md [hunk 7]
```diff
@@ -4,6 +4,51 @@ displayed_sidebar: "Chinese"
...
+- Hint 中支持设置用户自定义变量。[#40746](https://github.com/StarRocks/starrocks/pull/40746)
+- Hive Catalog 支持 CREATE TABLE LIKE。[#37685](https://github.com/StarRocks/starrocks/pull/37685) 
+- 新增 `information_schema.partitions_meta` 视图，提供丰富的 PARTITION 元信息。[#39265](https://github.com/StarRocks/starrocks/pull/39265)
+- 新增 `sys.fe_memory_usage` 视图，提供 StarRocks 的内存使用信息。[#40464](https://github.com/StarRocks/starrocks/pull/40464)
+
+### 行为变更
+
+- `cbo_decimal_cast_string_strict` 用于优化器控制 DECIMAL 类型转为 STRING 类型的行为。默认值是 `true`，即执行严格转换（按 Scale 截断补 `0`）。在历史版本中没有严格按照 DECIMAL 类型进行补齐，从而在 DECIMAL 与 STRING 类型进行比等时会产生不同效果。[#40619](https://github.com/StarRocks/starrocks/pull/40619)
+- Iceberg Catalog 的参数 `enable_iceberg_metadata_cache` 默认值改为 `false`。在 v3.2.1 到 v3.2.3 版本中，此参数统一默认为 `true`，即默认缓存 Iceberg 的元数据信息。在 v3.2.4 及以上版本中，如果 Iceberg Catalog 元数据服务为 HMS，此参数默认值变更为 `false`，如果元数据服务为 Glue，此参数默认值仍为 `true`。[#41826](https://github.com/StarRocks/starrocks/pull/41826)
+- 修改发起物化视图刷新任务的用户，从原本的 `root` 用户变成创建物化视图的用户，已有的物化视图不受影响。[#40670](https://github.com/StarRocks/starrocks/pull/40670)
```
Comment (evelynzhaojie) [inline docs/zh/release_notes/release-3.2.md:26]: 修改能发起物化视图刷新任务的用户 --- 加个 ”能“

**Review by evelynzhaojie — DISMISSED** [review]

**Review by amber-create — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Pull request merged**